packages/graphql-language-service-server/src/MessageProcessor.js

```diff
@@ -401,11 +401,20 @@ export class MessageProcessor {
     if (range) {
       position.line -= range.start.line;
     }
-    const result = await this._languageService.getAutocompleteSuggestions(
+    let result = await this._languageService.getAutocompleteSuggestions(
       query,
       position,
       textDocument.uri,
     );
+    if (textDocument.uri.endsWith('js')) {
+      // This ensures completion items are brought to the top of the list when inside a tagged template
+      result = result.map(item => {
+        return {
+          ...item,
+          sortText: '0',
```

Comment:
Use something like `sortText: '0' + item.label` so that there is a clear tie breaker

Reply:
Will do